recipes/PyCm/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pycm" %}
+{% set version = "7.5.1" %}
+{% set tag = "v7r5p1" %}
+
+package:
+   name: "{{ name }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: 427466add2f0c3095a4aaece857cd403de603eda1a3462367deae03a4fb27057
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Does this make more sense?

```suggestion
  skip: true  # [not linux]
```

---

Yes, I agree that makes more sense to put not.